### PR TITLE
feat: update docs for Base mainnet

### PR DESCRIPTION
**Pull request by yi-sun**

### CODE REVIEW

**Comment by ytham** [comment]
changed base to `staging`

File: docs/docs/introduction/README.md [hunk 1]
```diff
@@ -9,11 +9,9 @@ slug: /
 
 ![Axiom Logo](@site/static/img/axiom_horizontal.png)
 
-Axiom allows smart contracts to **trustlessly compute over the entire history of Ethereum**, including transactions and receipts. Developers can send on-chain queries into Axiom, which are trustlessly fulfilled with ZK-verified results sent in a callback to the developer's smart contract. This allows developers to build on-chain applications which access more data at a lower cost without adding additional trust assumptions.
+Axiom allows smart contracts to **trustlessly compute over the entire blockchain history of Ethereum**, including transactions and receipts. Developers can send on-chain queries into Axiom, which are trustlessly fulfilled with ZK-verified results sent in a callback to the developer's smart contract. This allows developers to build on-chain applications which access more data at a lower cost without adding additional trust assumptions.
```
Comment (ytham) [inline docs/docs/introduction/README.md:12]: do we want to say "of EVM chains" maybe?

Comment (yi-sun) [reply docs/docs/introduction/README.md]: Updated in a bit vaguer fashion

File: docs/sdk/typescript-sdk/axiom-circuit/axiom-subqueries/receipt-subquery.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ Receipt Subqueries provide data about a completed transaction and any logs/event
 The number of receipt subqueries allowed in a single client circuit depends on the size of the [transaction](./transaction-subquery) and receipt subqueries in the circuit.
 
 - Receipt subqueries are not supported for transactions with more than `400` logs or for transactions with any log containing more than `2048` bytes in the data field.
+- On Base, receipt subqueries are not supported for deposit or withdraw transactions.
```
Comment (ytham) [inline docs/sdk/typescript-sdk/axiom-circuit/axiom-subqueries/receipt-subquery.md:18]: Optional: do we just want to say that "On OP stack chains, system transactions are not supported" so we don't need to update this note for Optimism later?

File: docs/sdk/typescript-sdk/axiom-circuit/axiom-subqueries/transaction-subquery.md [hunk 3]
```diff
@@ -15,6 +15,7 @@ Transaction subqueries provide data about a transaction that was submitted on-ch
 The number of transaction subqueries allowed in a single client circuit depends on the size of the transaction and [receipt](./receipt-subquery) subqueries in the circuit.
 
 - Transaction subqueries are not supported for transactions with input data length greater than `330000` bytes or for transactions with RLP encoded access list length greater than `131072` bytes.
+- On Base, transaction subqueries are not supported for deposit or withdraw transactions.
```
Comment (ytham) [inline docs/sdk/typescript-sdk/axiom-circuit/axiom-subqueries/transaction-subquery.md:18]: Optional: same as above

**Review by ytham — APPROVED** [review]
LGTM w/ optional comments

File: docs/sdk/typescript-sdk/axiom-circuit/axiom-subqueries/receipt-subquery.md (hunk 2 above)
Comment (yi-sun) [reply docs/sdk/typescript-sdk/axiom-circuit/axiom-subqueries/receipt-subquery.md:18]: Let's do this once we have more than 1 OP stack chain (so readers don't get the impression we are on more than just Base).

**Pull request merged**